pyccl/core.py

```diff
@@ -312,12 +312,19 @@ class Cosmology(object):
             of state. Defaults to -1.
         wa (:obj:`float`, optional): Second order term of dark energy equation
             of state. Defaults to 0.
+        T_CMB (:obj:`float`): The CMB temperature today. The default of
+            ``None`` uses the global CCL value in
+            ``pyccl.physical_constants.T_CMB``.
         bcm_log10Mc (:obj:`float`, optional): One of the parameters of the
             BCM model. Defaults to `np.log10(1.2e14)`.
         bcm_etab (:obj:`float`, optional): One of the parameters of the BCM
             model. Defaults to 0.5.
         bcm_ks (:obj:`float`, optional): One of the parameters of the BCM
             model. Defaults to 55.0.
+        mu_0 (:obj:`float`, optional): One of the parameters of the mu-Sigma
```

Comment:
Why is this showing up as a new line here? Didn't the MG PR bring it in already?

Reply:
Nope this was missed.

Reply:
So how was it possible to call an MG cosmology from python then?

Reply:
This is a change to the doc string, not the object signature.